presto-main/src/test/java/com/facebook/presto/type/BenchmarkDecimalOperators.java

```diff
@@ -429,6 +608,10 @@ private SqlDecimal randomDecimal(DecimalType type)
             int maxBits = (int) (Math.log(Math.pow(10, type.getPrecision())) / Math.log(2));
             BigInteger bigInteger = new BigInteger(maxBits, random);
 
+            if (bigInteger.equals(ZERO)) {
+                bigInteger = ONE;
```

Comment:
I think this is because we don't want to divide by 0 in divide and modulo benchmarks

@arhimondr